sql/moz-fx-data-shared-prod/firefox_desktop/serp_categorization_unnested/view.sql

```diff
@@ -0,0 +1,74 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_desktop.serp_categorization_unnested`
+AS
+WITH flat AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    document_id,
+    CONCAT(document_id, '-', event_offset) AS event_id,
+    event.timestamp AS event_timestamp,
```

Comment:
Are you able to do anything with this event timestamp? It's an offset in seconds but I don't know if we have the base timestamp that it's offset from.

Reply:
True, and we have the glean_timestamp as well. I'll remove it